includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1919,10 +1920,23 @@ protected function check_response_for_errors( $response ) {
 					$response_code
 				);
 			} elseif ( isset( $response_body['error'] ) ) {
+				$response_body_error_code = $response_body['error']['code'] ?? null;
+				$payment_intent_status    = $response_body['error']['payment_intent']['status'] ?? null;
+
+				// Check for an 'amount_too_large' error code and a 'requires_capture' payment intent status.
+				// We redact the API error message to prevent prompting the merchant to contact Stripe support
+				// when attempting to manually capture an amount greater than what's authorized. Contacting support is unnecessary in this scenario.
+				if ( isset( $response_body_error_code ) && 'amount_too_large' === $response_body_error_code && 'requires_capture' === $payment_intent_status ) {
```

Comment:
To double-check: is `elseif ( isset( $response_body['error'] ) )` the correct branch for the redacting message? Asking as I don;t recall exactly if we should aim `$response_body['code']` or `$response_body['error']['code']`.

Reply:
Yes, we should look into $response_body['error'] because the error has this shape:

<img width="1519" alt="Screenshot 2023-11-02 at 10 56 56" src="https://github.com/Automattic/woocommerce-payments/assets/1553182/60cccdea-8a76-4732-923d-bc36a6a6e179">

Thanks for double checking.